jobs/rocoto/aerosol_init.sh

```diff
@@ -6,22 +6,28 @@ source "$HOMEgfs/ush/preamble.sh"
 # Source FV3GFS workflow modules
 source $HOMEgfs/ush/load_fv3gfs_modules.sh
 status=$?
-[[ $status -ne 0 ]] && exit $status
+if [[ $status -ne 0 ]]; then
+    exit $status
```

Comment:
Missing double quotes and curly braces around status variable in this script. Check all three instances.